community/py-sqlparse/APKBUILD

```diff
@@ -20,6 +21,12 @@ build() {
 	python3 setup.py build
 }
 
+check() {
+	cd "$builddir"
+	python2 setup.py test
+	python3 setup.py test
```

Comment:
This should be 
```diff
-       python2 setup.py test
-       python3 setup.py test
+       python2 -m pytest .
+       python3 -m pytest .
```

Reply:
I couldn't reproduce this locally, but changed it anyway.